helm/cluster-azure/templates/helmreleases/csi.yaml

```diff
@@ -55,3 +53,49 @@ spec:
     global:
       podSecurityStandards:
         enforced: {{ .Values.global.podSecurityStandards.enforced }}
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: {{ include "resource.default.name" $ }}-azurefile-csi-driver
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    cluster.giantswarm.io/description: "{{ .Values.metadata.description }}"
+  labels:
+    cluster-apps-operator.giantswarm.io/watching: ""
+    {{- if .Values.metadata.servicePriority }}
+    giantswarm.io/service-priority: {{ .Values.metadata.servicePriority }}
+    {{- end }}
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  suspend: false
+  releaseName: azurefile-csi-driver
+  targetNamespace: kube-system
+  storageNamespace: kube-system
+  chart:
+    spec:
+      chart: azurefile-csi-driver-app
+      # used by renovate
+      # repo: giantswarm/azurefile-csi-driver-app
+      version: v1.26.0-gs2
```

Comment:
This will be bumped after https://github.com/giantswarm/azurefile-csi-driver-app/pull/17